apps/src/aichat/aichatApi.ts

```diff
@@ -10,9 +17,12 @@ import {
   ChatEvent,
 } from './types';
 
+const ROOT_URL = '/aichat';
 const CHAT_COMPLETION_URL = '/aichat/chat_completion';
 const CHAT_CHECK_SAFETY_URL = '/aichat/check_message_safety';
 const LOG_CHAT_EVENT_URL = '/aichat/log_chat_event';
+const MAX_POLLING_TIME_MS = 45000;
```

Comment:
A bit of an arbitrary upper bound; in testing I've seen responses take as long as 25 seconds, and as per product, 30s is about the longest we want to wait. I increased to 45s to have a bit of buffer, but can also reduce it to be closer to the 30s maximum.